src/api.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+const articleAPI = axios.create({
+    baseURL: "https://eb-nc-news.onrender.com/api"
+})
+
+export const getArticles = () => {
+    return articleAPI
+        .get("/articles")
+        .then((response) => {
+            return response.data.articles
+        })
+        .catch((error)=>{
+            console.log(error)
+        })
```

Comment:
this catch block isn't needed, as you continue the promise chain in the component you can just let the catch block there be the only catch